jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/LogStreamTable.kt

```diff
@@ -72,16 +71,20 @@ class LogStreamTable(
         Disposer.register(this@LogStreamTable, logStreamActor)
         channel = logStreamActor.channel
 
-        component.verticalScrollBar.addAdjustmentListener {
-            if (logsTable.model.rowCount == 0) {
-                return@addAdjustmentListener
+        component.verticalScrollBar.addAdjustmentListener(object : AdjustmentListener {
+            var lastAdjustment = component.verticalScrollBar.minimum
+            override fun adjustmentValueChanged(e: AdjustmentEvent?) {
```

Comment:
don't think this is actually nullable

Reply:
I don't think it will ever actually be null but the Java interface doesn't have anything to enforce that so I'd prefer to be defensive